samigo/samigo-app/src/webapp/jsf/author/item/itemHeadings.jsp

```diff
@@ -14,6 +14,13 @@ for (i=0; i<document.links.length; i++) {
 document.links[newindex].onclick();
 }
 
+jQuery(document).ready(function($){
```

Comment:
The dollar here is JSP, right? What if somebody adds $.noConflict at a later date, earlier in the code? This seems a risky approach. Is this recommended?